openstack/cronus/templates/cronus/_config.yaml.tpl

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.cronus.enabled -}}
+cronus:
+  cacheSize: {{ .Values.cronus.cacheSize }}
+  billingCacheTTL: {{ .Values.config.billingCacheTTL }}
+  aws:
+    forwardUserAgent: {{ .Values.config.forwardUserAgent }}
+    allowedServices:
+      email: {{ .Values.config.allowedServices.email }}
+  listenAddr:
+    http: # default :5000
+    smtp: # default :1025
+  keystone:
```

Comment:
We'll probably use a technical user in future, therefore I suggest to parametrize all possible auth parameters, not only application credentials. Besides, if application credentials are not defined, this shouldn't cause an error.

Reply:
```
  listenAddr:
    http: :{{ .Values.cronus.port.http }} # default :5000
    smtp: :{{ .Values.cronus.port.smtp }} # default :1025
```